packages/react-core/src/components/TextInputGroup/examples/TextInputGroupWithStatus.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+  Button,
+  ValidatedOptions,
+  Flex,
+  FlexItem
+} from '@patternfly/react-core';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+
+export const TextInputGroupWithStatus: React.FunctionComponent = () => {
+  const [successValue, setSuccessValue] = React.useState('Success validation');
+  const [warningValue, setWarningValue] = React.useState('Warning validation with icon at start');
```

Comment:
Quick question about "with icon at start", is the icon meant to come before the input? It is currently showing after the input on the right for both the warning and error. It doesn't look like there's support for icon position in the code but the text here kind of implies that.

Reply:
Ah so the "icon at start:" is in reference to the search icon being at the start. I could update the verbiage to say something like, "...with custom non-status icon at start..." wdyt?